Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.h

```diff
@@ -255,6 +255,13 @@ class MediaPlayerPrivateGStreamer : public MediaPlayerPrivateInterface
         StreamCollectionChanged = 1 << 7
     };
 
+    enum class PlaybackRatePausedState {
+        ManuallyPaused, // Initialization or user explicitly paused.
+        RatePaused, // Pipeline was playing and rate was set to zero.
```

Comment:
Which state has priority? If we're both paused with `.pause()` and with `.rate = 0`, are we `ManuallyPaused` or `RatePaused`? Do we get the same state if we do `.pause()` first and `.rate = 0` later than if we do `.rate = 0` first and `.pause()` second?

Reply:
calvaris: `ManuallyPaused` has priority <-- should be in the comment